### PR TITLE
novel: Journey to the Center of the Earth (1867) initial review

### PR DESCRIPTION
**Pull request by paul-sheridan**
Curiously, although they spent months underground there was not one mention of how the managed to wash down there; only a passing mention of them washing their faces, hands and feet after making it back to the surface. Rating 5/5.

### CODE REVIEW

File: notes/stories/literature/novel-various.st.txt [hunk 1]
```diff
@@ -1,3 +1,73 @@
...
+Jules Verne
+
+:: References
+https://en.wikipedia.org/wiki/Journey_to_the_Center_of_the_Earth
+
+:: Ratings
+5 <paul>
+
+:: Choice Themes
+travel into the Earth [The whole story is about a trio of intrepid explorers who trek down old lava tubes to ultimately reach the center of the Earth.]
```
Comment (odinlake) [inline notes/stories/literature/novel-various.st.txt:36]: Strikes me that this theme applies also in https://www.themeontology.org/story.php?name=writing%3A%20Inferno%20(1320)

File: notes/stories/literature/novel-various.st.txt [hunk 2]
```diff
@@ -1,3 +1,73 @@
...
+Hollow Earth [Professor Lidenbrock and his party pointedly proved wrong the prevailing theory that the Earth had a central source of intense heat by the simple expedient of trekking down the lava tubes of an extinct volcano all the way to the center of the planet.]
+geology [Professor Lidenbrock and his party commented upon numerous geological features on their trek down to the center of the Earth: Lidenbrock (and to a lesser extent Alex) relatively dated the geological strata they were passing through and examined their fossil contents, discussed about various minerals, entertained two opposing theories of what lay beneath the Earth, etc.]
+academia [Professor Lidenbrock was a stereotypical oblivious German academic who was interested solely in advancing science by disproving the prevailing theory that the Earth had an intense source of heat at its center.]
+the scientist profession [The trained geologist Professor Lidenbrock and his nephew protégé Alex were in their element as they explored the bowels of the Earth.]
+human character stereotype [The party guide Hans was portrayed as a stoic Scandinavian to the extreme.]
```
Comment (odinlake) [inline notes/stories/literature/novel-various.st.txt:46]: I approve.

File: notes/stories/literature/novel-various.st.txt [hunk 3]
```diff
@@ -1,3 +1,73 @@
...
+risk taking vs. playing it safe [Professor Lidenbrock single-mindedly insisted that the party take big risks at every turn in an effort to reach the center of the Earth, while his nephew Alex constantly countered him with suggestions to take the cautious route - or else return home the same way they came.]
+
+:: Minor Themes
+cryptographic cypher [Professor Lidenbrock and Alex broke Arne Saknussemm's variant on a transposition cipher to read the 16th century Icelandic alchemist's encrypted message describing how he'd made it to the center of the Earth and back.]
+romantic love [Alex was mooning over his love interest, Gretchen.]
+female stereotype [Professor Lidenbrock goddaughter Gretchen was a stereotypical timid and obedient 19th century female.]
```
Comment (odinlake) [inline notes/stories/literature/novel-various.st.txt:52]: I don't think we should use this when someone confirms to contemporary norms.

File: notes/stories/literature/novel-various.st.txt [hunk 4]
```diff
@@ -1,3 +1,73 @@
...
+solitary confinement as a punishment [A lost and alone Alex wondered to himself as to how many prisoners had gone mad from being locked up in solitary confinement.]
+human in danger [Alex was lost alone for days in the bowels of the Earth.]
+despair [Alex gave himself up for dead after wondering around alone in the bowels of the Earth for days on end, thinking he'd never find Professor Lidenbrock and Hans again.]
+facing death [Days of wonder around alone in the bowels of the Earth left Alex thinking he was goner. Professor Lidenbrock and Alex thought they might well die at one point when they were in the process of being shot up out of a volcano.]
+perilous voyage at sea [The party got caught in a severe lightning storm while traversing a vast subterranean in their makeshift raft.]
+isolated prehistoric land [The party discovered an primeval forest deep inside the Earth. In its depths they were stunned to find a prehistoric humanoid more than twelve feet in height who was watching over a herd of mastodons.]
```
Comment (odinlake) [inline notes/stories/literature/novel-various.st.txt:64]: Is this not a major theme??

File: notes/stories/literature/novel-various.st.txt [hunk 5]
```diff
@@ -1,3 +1,73 @@
...
+living dinosaur [The party witnessed an Ichthyosaurus battle and defeat a Plesiosaurus while traversing a vast subsurface ocean on their makeshift raft.]
```
Comment (odinlake) [inline notes/stories/literature/novel-various.st.txt:65]: Ditto: the dinosaurs and such are what I vividly remember from reading the novel long ago!

**Review by odinlake — APPROVED** [review]
Some comments inline though..

File: notes/stories/literature/novel-various.st.txt (hunk 1 above)
Comment (paul-sheridan) [reply notes/stories/literature/novel-various.st.txt:36]: Excellent! And I see Wikipedia agrees <https://en.wikipedia.org/wiki/Subterranean_fiction>. Shall I leave you the honor?

File: notes/stories/literature/novel-various.st.txt (hunk 2 above)
Comment (paul-sheridan) [reply notes/stories/literature/novel-various.st.txt:46]: Hah

File: notes/stories/literature/novel-various.st.txt (hunk 3 above)
Comment (paul-sheridan) [reply notes/stories/literature/novel-various.st.txt:52]: I'm happy to take it out here.

But we should  clarify what the [https://themeontology.org/theme.php?name=female%20stereotype](female stereotype) theme is suppose to be. For example, [https://themeontology.org/theme.php?name=hysterical%20woman%20stereotype](hysterical woman stereotype) conforms to contemporary norms in 1950s stories, and it's been used in such stories without hesitation.

Also, for the record, it's not clear cut that the stereotype in question here conforms with the contemporary norms - just sayin'.

File: notes/stories/literature/novel-various.st.txt (hunk 4 above)
Comment (paul-sheridan) [reply notes/stories/literature/novel-various.st.txt:64]: While striking, it was only two or there out of 55 chapters by my count - so minor territory, I'd say.

File: notes/stories/literature/novel-various.st.txt (hunk 1 above)
Comment (odinlake) [reply notes/stories/literature/novel-various.st.txt:36]: Forgot to talk about this today. I wanted to discuss how it relates to the "journey into the underworld" theme we have in Inferno, before adding.

File: notes/stories/literature/novel-various.st.txt (hunk 3 above)
Comment (odinlake) [reply notes/stories/literature/novel-various.st.txt:52]: Hysterical woman was never a "norm". It was a popular stereotype in the 50s that is now likely seen as a sexist thing to point out.

Comment (odinlake) [reply notes/stories/literature/novel-various.st.txt:52]: And I mean if you think it was excessive then a "stereotypically timid woman" theme is conceivable. I just don't think that was the case in this 19:th century story seen from a 19:th century perspective. Also it's getting vague - we should focus on identifying stereotypes that are so on-the-nose that there can be little arguing about them.

File: notes/stories/literature/novel-various.st.txt (hunk 4 above)
Comment (odinlake) [reply notes/stories/literature/novel-various.st.txt:64]: Couldn't you say the conclusion was that they had discovered such a thing and returned to brag about it? Your decision either way, just seems odd to me.

File: notes/stories/literature/novel-various.st.txt (hunk 2 above)
Comment (odinlake) [reply notes/stories/literature/novel-various.st.txt:46]: To be honest I think this stereotype is too vague to identify consistently. But possibly being of this noble lineage myself I just don't realise how imposing we may seem to lesser mortals.

Comment (paul-sheridan) [reply notes/stories/literature/novel-various.st.txt:46]: o_0

File: notes/stories/literature/novel-various.st.txt (hunk 3 above)
Comment (paul-sheridan) [reply notes/stories/literature/novel-various.st.txt:52]: Let's take it out then.

File: notes/stories/literature/novel-various.st.txt (hunk 4 above)
Comment (paul-sheridan) [reply notes/stories/literature/novel-various.st.txt:63]: Well, I'll put the onus on others to read it and say different. It wasn't much featured. The point that was most sensational is that the Earth did not have a central source of heat as the prevailing theory of the day claimed.

Comment (paul-sheridan) [reply notes/stories/literature/novel-various.st.txt]: I knocked up **isolated prehistoric land** to major after revisiting the text. In short, they were technically roaming through the prehistoric region from Chapter 32 through Chapter 39 - not an unsignificant chuck of the book.

File: notes/stories/literature/novel-various.st.txt (hunk 1 above)
Comment (paul-sheridan) [reply notes/stories/literature/novel-various.st.txt:36]: Very good. Shall  we merge this story and take up the matter of what to do with Inferno over the weekend?

**Review by odinlake — APPROVED** [review]
Comments inline but merge if you are satisfied.

File: notes/stories/literature/novel-various.st.txt (hunk 2 above)
Comment (paul-sheridan) [reply notes/stories/literature/novel-various.st.txt:46]: How about using the general **human character** theme here? We used to have **stoicism** in there, but as things stand there's nothing to cover his nature of which much was made.

Comment (odinlake) [reply notes/stories/literature/novel-various.st.txt:46]: Perfect!

**Pull request merged**